openassessment/xblock/peer_assessment_mixin.py

```diff
@@ -165,7 +165,6 @@ def peer_path_and_context(self, continue_grading):
 
         context_dict = {
             "rubric_criteria": self.rubric_criteria_with_labels,
-            "estimated_time": "20 minutes",  # TODO: Need to configure this.
```

Comment:
This was never being used, so by the Boy Scout rule, I'm removing it.

Confirmation that it doesn't appear in any templates anywhere (run after making these changes):
```
ericfischer@Eric-Fischers-MacBook-Pro:~/devstack/src/edx-ora2$ grep -rnw . -e "estimated_time"
grep: ./edx-ora2/local/bin: No such file or directory
grep: ./edx-ora2/local/include: No such file or directory
grep: ./edx-ora2/local/lib: No such file or directory

ericfischer@Eric-Fischers-MacBook-Pro:~/devstack/src/edx-ora2$ 
```